studio-docs/source/org/members.mdx

```diff
@@ -5,53 +5,65 @@ sidebar_title: Members and roles
+| `Billing Manager`    | Has management access to organization-level configuration and billing. Can also remove members (but not invite them). No access to graphs.                                                                         |
+
+Only members with the `Organization Admin` role can assign roles to other members. You can see which role you have in a particular organization from your [user settings page](https://studio.apollographql.com/user-settings), or from the organization's Members tab.
+
+## Graph-specific member roles
+
+You can override a member's organization-wide role for individual graphs. For example, if a member has the `Observer` role in your organization, you can assign them the `Contributor` role for a graph that they need extra access to.
+
+> A member's graph-specific role must have _more_ permissions than their organization-wide role.
+
+Members with the `Graph Admin` or `Organization Admin` role can assign graph-specific roles from the Access tab of the graph's Settings page.
+
+## Role permissions
+
+> We recently added the `Graph Admin` role and reduced permissions for members with the `Contributor` role.
+
+| Action                                                 | Org Admin | Graph Admin | Billing Manager | Contributor | Observer | Consumer |
+| :----------------------------------------------------- | :-------: | :---------: | :-------------: | :---------: | :------: | :------: |
+| Invite members                                         |     ✓     |             |                 |             |          |          |
+| Remove members                                         |     ✓     |             |        ✓        |             |          |          |
+| View and edit billing information                      |     ✓     |             |        ✓        |             |          |          |
+| Manage organization configuration (name, avatar, etc.) |     ✓     |             |        ✓        |             |          |          |
+| Delete graphs                                          |     ✓     |      ✓      |                 |             |          |          |
+| Create, rename graphs                                  |     ✓     |      ✓      |                 |             |          |          |
+| Manage graph integrations (Datadog, Slack, etc.)       |     ✓     |      ✓      |                 |             |          |          |
+| View and manage graph API keys                         |     ✓     |      ✓      |                 |             |          |          |
+| Modify schema checks configuration                     |     ✓     |      ✓      |                 |             |          |          |
+| Push schemas to your graph                             |     ✓     |      ✓      |                 |      ✓      |          |          |
```

Comment:
I would consider just calling this line "Push schemas to a non-protected variant" (or "unprotected"? I'm not sure what vocab we are using)

We should include somewhere on the graph managing explorer settings, which is the same "contributor for unprotected" status. We should maybe also include reporting usage, or I can mention that separately in the text I'm going to add about how graph API keys relate.